src/style.css

```diff
@@ -1 +1,44 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin: 0 0 0 4px;
```

Comment:
What if we will edit code in the future? I thought it will be easier to edit margin than trying to find margin-right in the code. Just explaining my thoughts and actions, nothing more)